stardis/radiation_field/source_functions/blackbody.py

```diff
@@ -0,0 +1,35 @@
+import numpy as np
+from astropy import units as u, constants as const
+import numba
+
+H_CGS = const.h.cgs.value
+C_CGS = const.c.cgs.value
+K_B_CGS = const.k_B.cgs.value
+
+
+@numba.njit
+def bb_nu(tracing_nus, temps):
```

Comment:
I don't like this function name. But perhaps for a later PR

Reply:
maybe `blackbody_flux_nu` would be better. Also it won't cause confusion on whether the function returns flux or frequency.

Reply:
I agree that I don't like that the function name should be changed, but I was trying to leave the code as intact as possible for the parts that were simply getting moved around and not really altered. Might as well change it if we're already looking at it though. I'll adopt Sona's suggestion.